tests/test_requests.py

```diff
@@ -484,6 +484,7 @@ def test_BASICAUTH_TUPLE_HTTP_200_OK_GET(self, httpbin):
         'username, password', (
             ('user', 'pass'),
             (u'имя'.encode('utf-8'), u'пароль'.encode('utf-8')),
+            (42, 42)
```

Comment:
This should also have had a test with `(None, None)` just to cover all of our bases.